src/main/java/com/visenze/productsearch/README.md

```diff
@@ -147,7 +139,7 @@ ProductSearch api = new ProductSearch.Builder(APP_KEY, PLACEMENT_ID)
                      
 // Use the ProductSearch object to perform searches, it will return a
 // ProductSearchResponse object that contains all the searched results info 
-ProductSearchResponse searchResults = api.visualSimilarSearch(params);  
+ProductSearchResponse searchResults = api.recomendation(params);  
```

Comment:
looks like typo. should be `recommendation`? missing an `m`

Reply:
unfortunately it is the code typo, i can fix but then need to rename and may break backwards compatibility.

Reply:
ic ok

Reply:
https://github.com/visenze/visearch-sdk-java/blob/master/src/main/java/com/visenze/productsearch/ProductSearch.java#L211